extension/js/content_scripts/webmail/gmail_element_replacer.ts

```diff
@@ -125,11 +125,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
         const lastReplyBtn = $(convoReplyBtnsToReplaceArr.pop()!);
         $(lastReplyBtn).addClass('inserted');
         const element = $(this.factory.btnReply()).insertBefore($(lastReplyBtn).children().last());
-        if (isEncrypted) {
-          element.click(Ui.event.prevent('double', Catch.try(this.setReplyBoxEditable)));
-        } else {
-          element.click(Ui.event.prevent('double', Catch.try(() => this.replaceStandardReplyBox(true, true)))); // xss-safe-factory
-        }
+        element.click(Ui.event.prevent('double', Catch.try(this.setReplyBoxEditable)));
```

Comment:
Why this change? I think the `true, true` is a way to force replacement of the standard reply box even when the conversation is otherwise not encrypted (eg clicking "encrypted reply") on a regular conversation.